code/recipe/ingredients.md

```diff
@@ -1,3 +1,3 @@
-* 2 avacados
+* 2 avocados

```

Comment:
We want this typo here as it pertains to one of the exercises

Reply:
Oops, my bad! I'll revert.

Reply:
The spellchecker will freakout in every commit :P

Reply:
Actually this one was purely my mistake 🙃. I've set the spellchecker to only look at the `_episodes` folder because it was complaining about various typos in Carpentries files that we don't care about

Reply:
Ah, that's convenient!